### PR TITLE
[examples] Add SRE16 recipe.

### PR DESCRIPTION
**Pull request by czy97**

### CODE REVIEW

**Comment by wsstriving** [comment]
Will merge this after the plda implementation

File: tools/extract_embedding.sh [hunk 1]
```diff
@@ -24,6 +24,9 @@ store_dir=
 batch_size=1
 num_workers=1
 nj=4
+reverb_data=data/rirs/lmdb
+noise_data=data/rirs/lmdb
```
Comment (JiJiJiang) [inline tools/extract_embedding.sh:28]: todo: noise_data=data/musan/lmdb

Comment (czy97) [reply tools/extract_embedding.sh:28]: 👌🏻

File: examples/sre/v2/local/extract_sre.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+# limitations under the License.
+
+exp_dir=''
+model_path=''
+nj=4
+gpus="[0,1]"
+data_type="shard"  # shard/raw/feat
+data=data
+reverb_data=data/rirs/lmdb
+noise_data=data/rirs/lmdb
```
Comment (JiJiJiang) [inline examples/sre/v2/local/extract_sre.sh:25]: TODO: as below

File: examples/sre/v2/local/spk2utt_to_utt2spk.pl
Comment (JiJiJiang) [inline examples/sre/v2/local/spk2utt_to_utt2spk.pl:1]: TODO: move this file to tools

File: examples/sre/v2/local/utt2spk_to_spk2utt.pl
Comment (JiJiJiang) [inline examples/sre/v2/local/utt2spk_to_spk2utt.pl:1]: TODO: delete this file cause it's already in tools

File: examples/sre/v2/local/fix_data_dir.sh
Comment (JiJiJiang) [inline examples/sre/v2/local/fix_data_dir.sh:1]: TODO: move this file to tools

File: examples/sre/v2/local/filter_scp.pl
Comment (JiJiJiang) [inline examples/sre/v2/local/filter_scp.pl:1]: TODO: move this file to tools

**Comment by JiJiJiang** [comment]
Consider the above-mentioned update.

**Review by JiJiJiang — APPROVED** [review]
Well done!

**Pull request merged**